repositories.yaml

```diff
@@ -123,3 +123,26 @@ firefox-reality:
   dependencies:
     - org.mozilla.components:service-glean
     - org.mozilla.components:support-sync-telemetry
+lockwise-ios:
+  app_id: org.mozilla.ios.Lockbox
+  notification_emails:
+    - frank@mozilla.com
+    - tlong@mozilla.com
+  url: 'https://github.com/mozilla-lockwise/lockwise-ios'
+  dependencies:
+    - org.mozilla.components:service-glean
+firefox-desktop:
+  app_id: org.mozilla.fogotype
+  notification_emails:
+    - frank@mozilla.com
+    - chutten@mozilla.com
+  url: 'https://github.com/mozilla/gecko-dev'
+  dependencies:
+    - org.mozilla.components:service-glean
+lockwise-android:
+  app_id: mozilla.Lockbox
+  notification_emails:
+    - frank@mozilla.com
```

Comment:
I think we should add `lockwise-dev@mozilla.com` here as well

Reply:
Cool, added